workflow.yml

```diff
@@ -0,0 +1,74 @@
+openapi: 3.0.0
+info:
+  version: "0.0.3-unstable"
+  title: VC Workflow HTTP API
+  description: This is an Experimental Open API Specification for the [VC Data Model](https://www.w3.org/TR/vc-data-model/).
+  license:
+    name: W3C Software and Document License
+    url: http://www.w3.org/Consortium/Legal/copyright-software.
+  contact:
+    name: GitHub Source Code
+    url: https://github.com/w3c-ccg/vc-api
+paths:
+  /workflows/{name}/start:
+    post:
+      summary: Start a workflow.
+      description:
+        Starts a workflow with the given name. Optionally, additional information can be provided to the server in the request body that can help it fine tune what information to request from the client in order to continue with the workflow.
+      requestBody:
+        description: Optional and arbitrary POST data that the client might provide to the server to help it fine tune what information to request from the client in order to continue with the workflow.
+        content:
+          application/json:
+            schema:
+              oneOf:
+                - type: object
+                - type: string
```

Comment:
I string a mistake here? lets not offer more types than we need... increases implementation burden... especially in languages that respect types.

Reply:
Hrm, for `application/json` it is a mistake. Fixed in 1574f791143531155320cce458ecbae3bb5fc9a5.

What I was going for is trying to convey that any arbitrary post data is probably fine here. For example, it should be fine if someone wanted to post `application/octet-stream` or `application/x-www-form-urlencoded` if their workflows supported that. I almost added it into this PR, but didn't. We don't want to make posting content other than `application/json` illegal. That said, we can deal with that when someone requests it.